doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -160,51 +174,34 @@ list of assertion commands is given in :ref:`Assertions`. The command
    the statement. In other words ``Proof using e`` is equivalent to
    ``Proof using Type + e`` for any declaration expression ``e``.
 
-   .. cmdv:: Proof using {+ @ident } with @tactic
-
-      Combines in a single line :cmd:`Proof with` and :cmd:`Proof using`.
-
-      .. seealso:: :ref:`tactics-implicit-automation`
+   .. todo: "All" should be in the grammar...
 
    .. cmdv:: Proof using All
 
-      Use all section variables.
+      :n:`All` (as an :n:`@ident`)
```

Comment:
No need to keep the `(as an ident)` part once it appears in the documented syntax.